src/fhirdefs/load.ts

```diff
@@ -25,13 +25,20 @@ export async function loadDependency(
   FHIRDefs: FHIRDefinitions,
   cachePath: string = path.join(os.homedir(), '.fhir', 'packages')
 ): Promise<FHIRDefinitions> {
+  if (/(?=.*[A-Z])/.test(packageName)) {
```

Comment:
This is just checking that the `packageName` has any capital letters right? I think you can simplify it to:
```javascript
/[A-Z]/.test(packageName)
```
or am I missing something?

Reply:
No, you're not missing anything! The regex you suggested works just fine. Addressed in commit `855cd26`.